routes/shopItemById.js

```diff
@@ -15,7 +15,21 @@ shopItemById.post("/", (req, res, next) => {
 });
 
 shopItemById.delete("/", (req, res, next) => {
-
+    req.params.id = Number(req.params.id);
+    if (!isNaN(req.params.id)) {
+        const ifExist = req.database.shopItems.findIndex((item) => {
```

Comment:
I don't like the name of this variable. The name of a variable should reflect what it is. `ifExist` is the index of the item to be deleted (or -1 if the items does not exist).

I'd suggest to rename this to `itemIndex`, or `deleteIndex`, or something similar

Reply:
On it boss!